docs/_source/components-overview/pop_syn/binary_star.rst

```diff
@@ -22,98 +25,191 @@ BINARYPROPERTIES
 
 The binary properties are defined as follows
 
-.. csv-table:: BINARYPROPERTIES
-   :header: "Properties", "Descriptions"
-   :widths: 50, 150
-
-   `state`, "The state of the binary, see state options."
-   `event`, "The event of the binary, see event options."
-   `time`, "Age of the binary system in yr."
-   `separation`, "Orbital separation in R_sun."
-   `orbital_period`, "Orbital period in days."
-   `eccentricity`, "Orbital eccentricity."
-   `V_sys`, "Velocity of the centre of mass of the binary [Vx, Vy, Vz] in km/s."
-   `mass_transfer_case`, "Mass transfer case, see MT case options."
-   `lg_mtransfer_rate`, "the logarithm of the mass transfer rate in Msun/yr."
-   `step_names`, "Names of the steps in the evolution."
-   `step_times`, 'Time spend in the steps in the evolution.'
-   `rl_relative_overflow_1`, "The relative overflow of the Roche Lobe of star 1."
-   `rl_relative_overflow_2`, "The relative overflow of the Roche Lobe of star 2."
-   `trap_radius`, "The trapping radius of the binary in R_sun."
-   `acc_radius`, "The accretion radius of the binary in R_sun."
-   `t_sync_rad_1`, "?"
-   `t_sync_conv_1`, "?"
-   `t_sync_rad_2`, "?"
-   `t_sync_conv_2`, "?"
-   `nearest_neighbour_distance`, "The distance to the nearest neighbour for NN interpolation."
+.. list-table:: BINARYPROPERTIES
+  :header-rows: 1
+  :widths: 50 150
```

Comment:
I'm not sure, whether it is a good idea to use fixed table widths. This might cause displaying issues on devices with small screens. Have you checked, that it displays well on e.g. a smart phone?

Reply:
So this works fine with different sizes and automatically readjusts